### PR TITLE
Add listeners and attributes to tooltips

### PR DESCRIPTION
**Pull request by nicko-winner**
Add event listeners to tool tips

### CODE REVIEW

File: src/Tooltip/stories.js [hunk 1]
```diff
@@ -1,43 +1,31 @@
...
-
-    const handleChange = ({ target: { value } }) => {
-      setPosition(value);
-    };
+    let position = select('positions', POSITIONS, 'top-left');
+    let content = text('content', 'I am a tool tip');
 
     return (
-      <div>
+      <>
```
Comment (100stacks) [inline src/Tooltip/stories.js:18]: Is this different from https://github.com/HelixDesignSystem/helix-react/pull/116/files#r508772386?

**Review by 100stacks — CHANGES_REQUESTED** [review]

File: src/Tooltip/stories.js (hunk 1 above)
Comment (nicko-winner) [reply src/Tooltip/stories.js:18]: Yep they are different, tooltips do not seem to suffer from the same issues as Alerts. The other difference is this is a storybook story and not a wrapper. Issue seems to popup when the wrappers top level HTML node gets unexpectedly removed by helix.

Comment (100stacks) [reply src/Tooltip/stories.js:18]: Ok thanks for the explanation here also.  🙂

**Review by 100stacks — APPROVED** [review]
Dev LGTM

**Pull request merged**